force-app/main/default/classes/HH_Container_LCTRL.cls

```diff
@@ -615,8 +640,10 @@ public with sharing class HH_Container_LCTRL {
                 address.MailingState__c = (String)household.get('BillingState');
                 address.MailingPostalCode__c = (String)household.get('BillingPostalCode');
                 address.MailingCountry__c = (String)household.get('BillingCountry');
-                address.Undeliverable__c = (Boolean)household.get(
-                    UTIL_Namespace.StrTokenNSPrefix('Undeliverable_Address__c'));
+                if (canReadAddressUndeliverable && canReadAccountUndeliverable) {
```

Comment:
Does this check (and the ones below) also need for `canUpdate`?

Reply:
During my testing, I didn't see a case where the User can explicitly set the Undeliverable checkbox through this page. If changing an Address causes a record to be set as Undeliverable, that code should run without sharing via the TDTM trigger, not in the Manage Household logic.

Reply:
Interesting. If the page doesn't let the user modify the field value, why does the controller allow updating the field value?

Reply:
Ha, I walked away and thought the same thing. Since the Apex is updating values, the User will need that Perm as well. I can't speak to the "why", maybe some of this can be rewritten now that we're allowing use of the form without Undeliverable Address permissions, this first pass was just to get it working.